samples/README.md

```diff
@@ -13,9 +13,9 @@ kubectl create -f https://github.com/nirmata/kyverno/raw/master/definitions/inst
 
 ````bash
 
-kubectl create -f [samples/best_practices/](best_practices/)
+kubectl create -f [samples/best_practices/](best_practices)
```

Comment:
Can we provide complete links instead of relative links?

Reply:
This is in the bash, so link would not work in this case

Reply:
Ok, I got it!